inductiva/simulators/dualsphysics.py

```diff
@@ -2,36 +2,37 @@
 
 from typing import Optional, List
 
-from inductiva import types, tasks, resources
-from inductiva.simulators import Simulator
+from inductiva import types, tasks, simulators
 
 
-class DualSPHysics(Simulator):
+class DualSPHysics(simulators.Simulator):
     """Class to invoke a generic DualSPHysics simulation on the API."""
 
     def __init__(self):
         super().__init__()
         self.api_method_name = "sph.dualsphysics.run_simulation"
 
+    @simulators.simulator.mpi_disabled
```

Comment:
Afeter some thought, I came to the conclusion that I don't like have a decorator marking something that is not available. It should be marking something that is available. Let's talk about this